### PR TITLE
spark analysis

### PR DESCRIPTION
**Pull request by malvikaRavichandran**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -26,15 +26,14 @@ def result():
       gt_viz, df = generate_visualizations(df,channel_id)
       ml_start_time = time.time()
       ml_results = ml_classifiers(df,channel_id)
-      exe_time["pd_ml"]=(time.time()-ml_start_time)
+      exe_time["pd_ml"]=(round(time.time()-ml_start_time,2))
```
Comment (amoghkokari) [inline main.py:29]: good

File: main.py [hunk 2]
```diff
@@ -26,15 +26,14 @@ def result():
...
       spark_start_time = time.time()
       ml_spark = spark_ml.spark_main(channel_id)
-      exe_time["spark_ml"]=(time.time()-spark_start_time)
-      print(ml_spark)
-      print(exe_time)
+      exe_time["spark_ml"]=(round(time.time()-spark_start_time,2))
```
Comment (amoghkokari) [inline main.py:32]: nice

File: main.py [hunk 3]
```diff
@@ -26,15 +26,14 @@ def result():
...
+      # print(ml_spark)
+      # print(exe_time)
```
Comment (amoghkokari) [inline main.py]: please remove this

File: templates/result.html [hunk 4]
```diff
@@ -34,7 +34,7 @@ <h1>Word Clouds on all the video data present in the Channel</h1>
 			background: #ffffff;
 			color: #313030;
 		}
-	</style>
+	</style>
```
Comment (amoghkokari) [inline templates/result.html]: remove extra space

**Review by amoghkokari — CHANGES_REQUESTED** [review]
please address the comments

File: templates/result.html (hunk 4 above)
Comment (malvikaRavichandran) [reply templates/result.html]: done

File: main.py (hunk 3 above)
Comment (malvikaRavichandran) [reply main.py]: ok

**Review by amoghkokari — APPROVED** [review]
well done

**Pull request merged**